app/javascript/packages/document-capture-polling/index.js

```diff
@@ -40,6 +41,10 @@ export class DocumentCapturePolling {
     this.toggleFormVisible(false);
     this.trackEvent('IdV: Link sent capture doc polling started');
     this.schedulePoll();
+    DocumentCapturePolling.bindPromptOnNavigate(true);
```

Comment:
I'm a bit conflicted on this [`class-methods-use-this` rule](https://eslint.org/docs/rules/class-methods-use-this) we include through the Airbnb ESLint preset. I do think it can sometimes hint at an opportunity for valid refactoring (extracting function, or static function), but most often I find it's more heavy-handed than it needs to be.

Reply:
Yeah this seems like a silly lint rule, I would totally be in favor of making this an instance method

Reply:
> Yeah this seems like a silly lint rule, I would totally be in favor of making this an instance method

👍  Updated in 55cde74 + 921a651